core/timeline_inc.php

```diff
@@ -47,6 +47,12 @@
 $t_block_icon = $t_collapse_block ? 'fa-chevron-down' : 'fa-chevron-up';
 
 $t_url_page = string_sanitize_url( basename( $_SERVER['SCRIPT_NAME'] ) );
+# Timeline shows buttons to reload the page with new timeline parameters
+# we must preserve parent script query parameters
+$t_url_params = $_GET;
```

Comment:
I thought the same...
But in that scenario, i think you could argue that the GET request is a viewing action and is safe to be repeated, while POST implies modification. The similar idea on why post usually is not cached.